### PR TITLE
59 report tlf

### PR DESCRIPTION
**Pull request by laxamanaj**
I'm working with with @robertdevine and @pstata to look into this report-tlf.pdf generation per #59 .  The goal we are trying to accomplish is to output the same report-tlf.pdf as pilot 1, here : https://github.com/RConsortium/submissions-pilot1-to-fda/blob/main/m1/us/report-tlf.pdf 
The updates I want to show is that we included the timestamps in the footnote, showing that we did rerun these for pilot 3 at the time of submission.

When you pull this branch, you'll see I started an report-tlf-pilot3.Rmd here : `/submissions/programs/report-tlf-pilot3.Rmd`.  Right now as it runs the demog table and the kmplot seem to come out ok.  The demog table could use some minor adjusting so the header lines don't wrap over.

As you see in the `/submissions/programs/report-tlf-pilot3.Rmd`, I want to be able to source our first line programs from our `programs` folder to output in one markdown pdf file so that we keep the program files as the single source of truth.

When running this .Rmd, the `report-tlf-pilot3.pdf` will save in the programs folder. At the same time it also runs all of the table programs and updates them in the output folder. In this way the timestamps will always match up in the output folder and the final `report-tlf-pilot3.pdf`

The tables primary and efficacy, I am having a bit of trouble outputting with the titles and footnotes as they show in their original .rtf outputs.  If someone can help look into this to bring this `report-tlf.pdf` similar to pilot 1, that is the goal.

to summarize fixes needed to show in this R markdown : 
1. need to fix demog table so headers lines don't wrap over (e.g. decrease size of table maybe?)
2. primary and efficacy tables are currently outputting in .rtf formats, with titles and footnotes. Need to figure out a way to keep formatting from .rtf to stay the same in this report-tlf.pdf, with title and footnotes. especially the time stamps.
3. have  `report-tlf-pilot3.pdf` get saved in our `output` folder programmatically (e.g. maybe there is r markdown `output=[file path]` argument we can add)?.
4. will update the batch run script to also include the run of the R markdown

I plan to meet with Robert and Phani tomorrow, but want to tag in @DeclanHodges @thomas-neitmann in this review to see if they have any ideas to resolve the issues mentioned above, since they'll be logging in soon in EU.

Thanks in advance.
Joel

### CODE REVIEW

**Comment by bms63** [comment]
is this still being worked on?

**Comment by laxamanaj** [comment]
Yes. I'll need to generate this after the final run.

**Comment by robertdevine** [comment]
Hi @laxamanaj - please review. As noted the use of \includegraphics does address the issue you raised at last Friday's Check-In. Thanks.

_Summary_:
Polishing report-tlf-pilot3 with includegraphics so the RTF.PDF outputs display on the same page under the Table 14-3.01 and (sequentially with pagination) Table 14-3.02 headers. Set KM Plot fig.align and out.width r plot property values. Modified footers for KM Plot and Demographic Table to comply with the June 2023 FDA Guidelines.

File: submission/programs/report-tlf-pilot3.Rmd [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+#### Table 14-3.02
+#### Primary Endpoint Analysis: Glucose (mmol/L) - Summary at Week 20 - LOCF
+
+```{r efficacy, echo=FALSE, message=FALSE, comment=NA}
+# setwd("/cloud/project")
+# source("/cloud/project/submission/programs/tlf-efficacy.r")
+# print(tbl)
+```
+
+\includegraphics[page=1]{/cloud/project/submission/output/tlf-efficacy-pilot3.rtf.pdf}
```
Comment (laxamanaj) [inline submission/programs/report-tlf-pilot3.Rmd:59]: Hi, @robertdevine .  When I pulled this branch and ran the .Rmd, Table 14-3.02 was off center in the report-tlf-pilot3.pdf that I generated.  Is this the same with yours? See below.
image

**Review by laxamanaj — COMMENTED** [review]
Added comment regarding Table 14-3.02 output.

**Review by robertdevine — APPROVED** [review]
Thank you @laxamanaj

**Comment by laxamanaj** [comment]
closes #59

**Pull request merged**